src/log-pcap.c

```diff
@@ -1307,6 +1345,10 @@ static OutputInitResult PcapLogInitCtx(ConfNode *conf)
     int en;
     PCRE2_SIZE eo = 0;
 
+    if (g_pcap_data) {
+        FatalError("A pcap-log instance is already active, only one can be enabled.");
```

Comment:
did this work before? If so it's a limitation we need to add to the upgrade doc

Reply:
It does crash, but that might only be on exit.